angular/src/components/export.component.ts

```diff
@@ -123,6 +129,38 @@ export class ExportComponent implements OnInit {
         await this.eventService.collect(EventType.User_ClientExportedVault);
     }
 
+    protected async verifySecret(): Promise<boolean> {
+        const verification: Verification = this.exportForm.get('secret').value;
+        if (verification?.secret == null || verification.secret === '') {
+            this.platformUtilsService.showToast('error', this.i18nService.t('errorOccurred'),
+                this.i18nService.t('verificationCodeRequired'));
```

Comment:
Do we need to use `invalidMasterPassword` here if not using crypto agent?

Reply:
As discussed, we'll rely on the HTML5 `required` attribute for form validation. That is already implemented in the `VerifyMasterPasswordComponent` template.

This means that the error here does not have to be gracefully handled or user friendly, as it should never happen. I've changed it to just `throw` a non-translated message for debugging purposes.